Plugin/ProductList/Item.php

```diff
@@ -0,0 +1,78 @@
+<?php
+declare(strict_types=1);
+
+namespace Bazaarvoice\Connector\Plugin\ProductList;
+
+use Bazaarvoice\Connector\Api\ConfigProviderInterface;
+use Bazaarvoice\Connector\Api\StringFormatterInterface;
+
+/**
+ * Class Item
+ *
+ * @package Bazaarvoice\Connector\Plugin\ProductList
+ */
+class Item
+{
+    /* @var \Magento\Catalog\Api\Data\ProductInterface|\Magento\Catalog\Model\Product */
+    protected $product;
+    /**
+     * @var
+     */
+    protected $productIds;
+    /**
+     * @var
+     */
+    protected $type;
+    /**
+     * @var ConfigProviderInterface
+     */
+    protected $configProvider;
+    /**
+     * @var StringFormatterInterface
+     */
+    protected $stringFormatter;
+
+    /**
+     * Item constructor.
+     *
+     * @param ConfigProviderInterface  $configProvider
+     * @param StringFormatterInterface $stringFormatter
+     */
+    public function __construct(
+        ConfigProviderInterface $configProvider,
+        StringFormatterInterface $stringFormatter
+    ) {
+        $this->configProvider = $configProvider;
+        $this->stringFormatter = $stringFormatter;
+    }
+
+    // @codingStandardsIgnoreStart
+    public function beforeGetProductPrice(
```

Comment:
If you are implementing a "before" and "after", why not use an "around"?

Reply:
I do not see a reason why not. This code was originally written by StoreFront. I have added a todo to test combining these into an around plugin.